src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/PrisonerSearchService.kt

```diff
@@ -50,6 +51,7 @@ class PrisonerSearchService(
     return query?.let {
       val searchSourceBuilder = SearchSourceBuilder().apply {
         query(query.withDefaults(searchCriteria))
+        size(RESULT_HITS_MAX)
```

Comment:
@Mjwillis This seems reasonable to me for all searches but did you have any intent for the general searches to bring back only 10 offenders?

Reply:
@andymarke no it was a missing bit on my part.  I notice that UoF only returns 10 results which is not strictly correct!